commands/backfill.js

```diff
@@ -66,13 +66,22 @@ module.exports = function container (get, set, clear) {
             var opts = {product_id: product_id}
             if (mode === 'backward') {
               opts.to = marker.from
+              // bitfinex v2 api required timestamp
+              if (exchange_id === 'bitfinex' && !marker.from){
```

Comment:
We will really need to find a different solution for this, logic for exchanges should be limited to exchanges, and never live outside of that.